src/defaultContent/styles/manage-keys.css

```
@@ -98,7 +91,8 @@
 }
 
 .accordion-body {
-    padding: 1.25rem;
+    padding-left: 0;
+    padding: 1.25rem 1.25rem 1.25rem 0;
```

Biome (2.1.2)

[error] Unexpected shorthand property padding after padding-left



(lint/suspicious/noShorthandPropertyOverrides)